external/llvm-project/mlir/lib/Conversion/GPUToROCDL/LowerGpuOpsToROCDLOps.cpp

```diff
@@ -107,7 +109,8 @@ struct LowerGpuOpsToROCDLOpsPass
     // Keep last to prioritize newly-added type conversions, just in case
     // Note that since these patterns touch LLVM ops, they'll need to run after
     // conversion
-    populateBF16ToROCDLConversionPatterns(converter, bf16fixupPatterns);
+    mlir::LLVM::populateBF16ToLLVMConversionPatterns(converter,
```

Comment:
Let's not do it this way. Instead, use `NestedPassManager` (I think I remember the name correctly) to run the pass after the conversion call